### ISSUE TITLE
With regards to a perceptual loss

### ISSUE THREAD

**Issue by PetrochukM** [magenta/ddsp#18]
Posting this again because the issue was closed before my question was resolved: https://github.com/magenta/ddsp/issues/12

-----

@jesseengel Thanks for the response! 

You didn't quite answer my second question. Let me ask it in a different way...

With regards to the autoencoder configuration...
https://github.com/magenta/ddsp/blob/cd9811675873dc3f5c5fb3a51b5a48923b3330ae/ddsp/training/gin/models/ae.gin#L40

Is there a benefit to training on an amplitude spectrogram? It looks like it's also used in the loss in addition to the log scaled spectrogram.

The amplitude spectrogram is not meaningful from a human perspective, right?

**Comment by jesseengel** [comment]
Hi! Thanks for the more specific follow up. From a loss function perspective, both mag and logmag should be small for audio that is similar, but they emphasize different things. Logmag will put more weight on the quiet sections (since it squashes peaks with a log) while mag will put a lot more weight on peaks. Human hearing has a sensitivity that scales more logarithmically, but neither is a perfect proxy. We found in initial experiments that having both made optimization  easier / faster for our experiments, but it's not a hard rule by any means. So please do feel free to experiment and find what works best for you. 

I'll let you close this if you feel that has answered your question.

**Comment by PetrochukM** [comment]
Thanks! That answers it!
